tools/c7n_mailer/c7n_mailer/deploy.py

```diff
@@ -38,9 +38,19 @@ def dispatch(event, context):
 """
 
 
+def hack_ruamel():
+    # The way ruamel is installed it has no __init__.py(!). import finds it(!!)
+    # but imp.find_package doesn't(!!!). Hack it!
+    import ruamel
+    init_py = os.path.join(list(ruamel.__path__)[0], '__init__.py')
+    if not os.path.exists(init_py):
+        open(init_py, 'w+')
+
+
 def get_archive(config):
+    hack_ruamel()
     archive = PythonPackageArchive(
-        'c7n_mailer', 'ldap3', 'pyasn1', 'jinja2', 'markupsafe', 'yaml',
+        'c7n_mailer', 'ldap3', 'pyasn1', 'jinja2', 'markupsafe', 'ruamel',
```

Comment:
why the switch on ruamel from yaml.. afaics pyyaml is fine .. ie. https://travis-ci.org/yaml/pyyaml

Reply:
Because of https://github.com/capitalone/cloud-custodian/pull/1546#issuecomment-332479768, no?

I.e., PyYAML has a split codebase between 2 and 3. ruamel folds down into single codebase and passes PyYAML's test suite.

Reply:
ah the install time causes a run / config mismatch gotcha.